frontends/web/src/utils/markup.tsx

```diff
@@ -47,23 +47,16 @@ export function SimpleMarkup({ tagName, markup, ...props }: MarkupProps) {
 }
 
 /**
- * **multilineMarkup** splits a text by newline and renders each
+ * **<MultilineMarkup>** splits a text by newline and renders each
  * line with the `<SimmpleMarkup>` component.
- * **Note**: With current Preact 8 document fragments are not
- * supported so this has to be used as function, but once we move
- * to modern Preact/React we can use it as a normal component.
- * ### Example:
- * ```jsx
-    {multilineMarkup({ tagName: 'p', markup: 'messages\n\nwith <strong>newlines</strong>'})}
- * ```
  */
-export function multilineMarkup({ tagName, markup, ...props }: MarkupProps) {
-  return markup.split('\n').map((line: string, i: number) => (
-    SimpleMarkup({
-      key: `${line}-${i}`,
-      tagName,
-      markup: line,
-      ...props
-    })
-  ));
-}
+
+export const MultilineMarkup: FunctionComponent<MarkupProps> = ({ tagName, markup, ...props }) => {
```

Comment:
Is FunctionComponent correct here? I think @thisconnect once told me that we should only use it if there are children (or not even then?).

See also https://medium.com/raccoons-group/why-you-probably-shouldnt-use-react-fc-to-type-your-react-components-37ca1243dd13

I might be wrong and am mainly here to learn :nerd_face:

Reply:
yeah we use FunctionComponent everywhere at the moment and probably shouldn't. Lets address this in a different PR and update the whole codebase.